lib/node_modules/@stdlib/stats/base/dists/planck/logpmf/README.md

```diff
@@ -101,7 +101,7 @@ y = mylogpmf( 1.0 );
 
 ## Notes
 
--   In virtually all cases, using the `logpmf` or `logcdf` functions is preferable to manually computing the logarithm of the `pmf` or `cdf`, respectively, since the latter is prone to overflow and underflow.
+In virtually all cases, using the `logpmf` or `logcdf` functions is preferable to manually computing the logarithm of the `pmf` or `cdf`, respectively, since the latter is prone to overflow and underflow.
```

Comment:
```suggestion
-   In virtually all cases, using the `logpmf` or `logcdf` functions is preferable to manually computing the logarithm of the `pmf` or `cdf`, respectively, since the latter is prone to overflow and underflow.
```